jet_tracking/mpi_scripts/run_mpi_script

```diff
@@ -46,6 +47,11 @@ PROC=${PROC:='2'}
 #source /reg/g/psdm/sw/conda1/manage/bin/psconda.sh
 #cd /cds/group/pcds/epics-dev/aegger/jet_tracking/jet_tracking/mpi_scripts
 #pythonpathmunge '/cds/group/pcds/epics-dev/aegger/jet_tracking/jet_tracking'
-CONFIG=${CONFIG:=$MYDIR/jt_configs/xcs_config.yml}
+#CONFIG=${CONFIG:=$MYDIR/jt_configs/xcs_config.yml}
+CONFIG=${CONFIG:=$MYDIR/jt_configs/mfx_epix_config.yml}
+
+echo $CONFIG
 
 `which mpirun` -n $PROC python -u $MYDIR/mpi_scripts/mpi_driver.py --cfg_file $CONFIG
```

Comment:
Why does it need the `which`?

Reply:
Good question. Probably some legacy stuff from Adam. I've actually seen that syntax in quite a few places, but never understood the point of it...